applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS
+	bool "DVFS module"
+	depends on SOC_NRF54H20_CPUAPP
+	depends on BT_MAX_CONN=1
```

Comment:
`depends on CAF_BLE_COMMON_EVENTS || (BT_MAX_CONN = 1)`

This condition does not need to be met for configurations without Bluetooth LE, you added ifdefs to allow building in such configuration already

Reply:
Looking at this it seems to make sense to introduce module-specific options that would control handling given types of application events (`CONFIG_DESKTOP_DVFS_STATE_USB`, `CONFIG_DESKTOP_DVFS_STATE_LLPM`). The options could depend on required application events (`CONFIG_DESKTOP_DVFS_STATE_LLPM` would also depend on proper connection count) and they could be enabled by default.

Thanks to that Kconfig would clearly show dependencies, user will gain fine-grained control over the module and the module could be used also on dongles (only for USB part).

Reply:
If we don't want to let user control subset of handled events, we could make the options promptless

Reply:
will take a look under this task: https://nordicsemi.atlassian.net/browse/NCSDK-27381